app_dart/bin/server.dart

```diff
@@ -90,6 +90,14 @@ Future<void> main() async {
       '/api/github-webhook-pullrequest': GithubWebhook(
         config: config,
         pubsub: const PubSub(),
+        secret: config.webhookKey,
+        topic: 'github-webhooks',
+      ),
+      '/api/github/frob-webhook': GithubWebhook(
+        config: config,
+        pubsub: const PubSub(),
+        secret: config.frobWebhookKey,
+        topic: 'frob-webhooks',
```

Comment:
Do you see this new topic consuming more than just the re-request event in the future? Just curious on why we are making a new one instead of making the github-webhooks process the rerequest

Reply:
The new topic should have all events for the Google Testing app, which may differ from the flutter-dashboard app based on their configs. Both should have the pull request events.

Can you clarify "making the github-webhooks process the rerequest"? I went with this approach as having the GitHub webhooks handler process webhooks from multiple apps would require more work.

Reply:
> I went with this approach as having the GitHub webhooks handler process webhooks from multiple apps would require more work.

This pretty much answer my question. Thank you